### PR TITLE
Add my name

### PR DESCRIPTION
**Pull request by Hamidmirzapour**
@dga check and accept my pull request to also add my name to list and contribute to this amazing project

### CODE REVIEW

File: Submissions/hamid.json
Comment (ProsperousHeart) [inline Submissions/hamid.json:1]: This file should be removed

File: Submissions/Hamidmirzapour.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "hamid Mirzapour",
```
Comment (ProsperousHeart) [inline Submissions/Hamidmirzapour.json:2]: Should `hamid` be capitalized?

**Review by ProsperousHeart — CHANGES_REQUESTED** [review]
Please remove the file that is not the one you want. Whichever want you want, be sure the title matches your GitHub username:  `Hamidmirzapour`

File: Submissions/Hamidmirzapour.json (hunk 1 above)
Comment (Hamidmirzapour) [reply Submissions/Hamidmirzapour.json:2]: Sure, Thanks.

Comment (Hamidmirzapour) [reply Submissions/Hamidmirzapour.json]: In my last commit, I did what you want. Please pull my request. 
Thanks a lot.

**Review by ProsperousHeart — APPROVED** [review]
Approved!

**Pull request merged**